lib/modules/onc_program_procedure.yml

```diff
@@ -0,0 +1,1219 @@
+procedure:
+  - section: Paragraph (g)(10)(iii) - Application registration
+    steps:
+      - group: Application Registration
+        index: 1
+        SUT: |
+          The health IT developer demonstrates the Health IT Module
+          supports application registration with an authorization server for
+          the purposes of Electronic Health Information (EHI) access for
+          single patients, including support for application registration
+          functions to enable authentication and authorization in §
+          170.315(g)(10)(v)
+        TLV: |
+          The tester verifies the Health IT Module supports application
+          registration with an authorization server for the purposes of EHI
+          access for single patients, including support for application
+          registration functions to enable authentication and authorization in
+          § 170.315(g)(10)(v).
+        inferno_supported: no
+        inferno_notes: |
+          Application registration is considered out of scope for Inferno
+          Program Edition because it cannot be done in an automated fashion.
+          Inferno Program Edition may in the future include Dynamic Registration
+          functionality for those systems that support that standard.  It may
+          also include an input field to allow the tester to document that this
+          has been performed.
+      - index: 2
+        SUT: |
+          The health IT developer demonstrates the Health IT Module
+          supports application registration with an authorization server for
+          the purposes of EHI access for multiple patients, including support
+          for application registration functions to enable authentication and
+          authorization in § 170.315(g)(10)(v).
+        TLV: |
+          The tester verifies the Health IT Module supports application
+          registration with an authorization server for the purposes of EHI
+          access for multiple patients including support for application
+          registration functions to enable authentication and authorization in
+          § 170.315(g)(10)(v).
+        inferno_supported: no
+        inferno_notes: |
+          Application registration is considered out of scope for Inferno
```

Comment:
Since we're not requiring Dynamic Registration as part of the rule, I'm hesitant to say that we may include Dynamic Registration in the Program Edition without other caveats. We could word it as "Inferno Program Edition may include an optional test for conformance to the RFC 7591 'OAuth 2.0 Dynamic Client Registration Protocol Dynamic Registration' as an optional way for the health IT developer to demonstrate the Health IT Module supports application registration."